src/baseStyleUtils.js

```diff
@@ -0,0 +1,66 @@
+const css = (strings, ...expressions) => {
+  let str = ''
+  strings.forEach((string, i) => {
+    str += string + (expressions[i] || '')
+  })
+
+  return str
+}
+
+const generateBaseStyles = ({ breakpoint = '40em' }) => css`
+.super-responsive-table__th_pseudo {
+  display: none;
+}
+
+@media screen and (max-width: ${breakpoint}) {
+  .super-responsive-table__table,
+  .super-responsive-table__thead,
+  .super-responsive-table__tbody,
+  .super-responsive-table__th,
+  .super-responsive-table__td,
+  .super-responsive-table__tr {
+    display: block;
+  }
+
+  .super-responsive-table__thead {
+    position: absolute;
+    top: -9999px;
+    left: -9999px;
+  }
+
+  .super-responsive-table__td_pivoted {
+    position: relative;
+    margin-left: 50%;
+  }
+
+  .super-responsive-table__th_pseudo {
+    box-sizing: border-box;
+    display: block;
+    position: absolute;
+    top: 0;
+    left: -100%;
+    width: 100%;
```

Comment:
Set as such instead of something like `left: 0; width: 50%;` because we used `margin` instead of `padding` in the `td` above:
https://github.com/ua-oira/react-super-responsive-table/blob/5cb113ca5171c844f314a89bb08407ba2e652e77/src/baseStyleUtils.js#L31-L33